src/components/WwwFrame/WwwPage.vue

```diff
@@ -64,17 +79,32 @@ export default {
 		};
 	},
 	apollo: {
-		preFetch(config, client, { route }) {
+		preFetch(config, client, { cookieStore, route }) {
+			const getUserData = cookieStore.get(hasLentBeforeCookie) === undefined || cookieStore.get(hasDepositBeforeCookie) === undefined; // eslint-disable-line max-len
```

Comment:
What happens when these cookies are `false`, then the user does these actions, will these cookies always be `false`?

Reply:
If cookie is false, it will be handle by the Optimizely experiment. Added an image for reference.

![User Metrics: Audiences - Optimizely 2022-08-01 19-48-58](https://user-images.githubusercontent.com/102815442/182268551-67afef07-5149-461f-99e1-e09bbffd78bf.png)

Reply:
Right, but do we want the cookie value to change once the user takes an action, such as lend, where `hasLentBeforeCookie` would change from `false` to `true`?